doc/whats_new/v0.20.rst

```diff
@@ -188,6 +189,14 @@ Misc
   one was added (:ref:`environment_variable`).
   :issue:`11166`by `Gael Varoquaux`_
 
+Datasets
+
+- Added :func:`datasets.fetch_openml` to fetch datasets from `OpenML <http://openml.org>`.
+
+OpenML is a free, open data sharing platform and will be used instead of mldata
```

Comment:
Can you put this back on the previous line and aligned with the rest?

Reply:
Done